src/server/handlers/wallet/export.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: MIT
+//! wallet handlers and parameters
+
+use crate::identity::json::KeyInfoJson;
+use crate::identity::Wallet;
+use crate::server::JsonRPCRequestHandler;
+use async_trait::async_trait;
+use fvm_shared::address::Address;
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+use std::sync::{Arc, RwLock};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct WalletExportParams {
```

Comment:
not sure if this will pose some security risks as it's exposed over json rpc

Reply:
Good point, is definitely an issue without proper authentication. I will probably make it available only through the CLI in the next PR, so only someone with access to the local process can export the keys. Thanks!